### ISSUE TITLE
INT false negatives for the plural argument of `ngettext`

### ISSUE THREAD

**Issue by dscorbett** [astral-sh/ruff#21062]
### Summary

[`f-string-in-get-text-func-call` (INT001)](https://docs.astral.sh/ruff/rules/f-string-in-get-text-func-call/), [`format-in-get-text-func-call` (INT002)](https://docs.astral.sh/ruff/rules/format-in-get-text-func-call/), and [`printf-in-get-text-func-call` (INT003)](https://docs.astral.sh/ruff/rules/printf-in-get-text-func-call/) only check the first argument (singular) of `ngettext` but skip the second (plural). The reasons to avoid f-strings etc. in the first argument apply equally to the second. This is true whether or not `ngettext` is imported, and if so whether it is aliased. [Example](https://play.ruff.rs/53b13c7c-340d-4f64-9e24-59ed0831d2ef):
```console
$ cat >int00.py <<'# EOF'
from gettext import ngettext
def f(n):
    ngettext("one", f"{n}", n)
    ngettext("one", "{}".format(n), n)
    ngettext("one", "%d" % n, n)
# EOF

$ ruff --isolated check int00.py --select INT001,INT002,INT003
All checks passed!

$ sed '/import/d' int00.py | ruff --isolated check - --select INT001,INT002,INT003
All checks passed!

$ sed '/import/s/$/ as n_/; s/ngettext(/n_(/' int00.py | ruff --isolated check - --select INT001,INT002,INT003
All checks passed!
```

### Version

ruff 0.14.2 (83a3bc4ee 2025-10-23)

**Comment by ntBre** [comment]
I think it makes sense to expand the rules to check the second argument in preview.
